cmd/clusterctl/README.md

```diff
@@ -87,10 +87,10 @@ Also, some environment variables are supported:
 
 ### Interacting with your cluster
 
-If you are using kind, set the `KUBECONFIG` environment variable first before using kubectl:
+If you are using kind, set your context to point at the newly created cluster:
 
 ```
-export KUBECONFIG="$(kind get kubeconfig-path --name="clusterapi")"
+kubectl cluster-info --context kind-clusterapi
```

Comment:
Do we need to expose `KUBECONFIG` first?

Reply:
@gyliu513 Don't think that is required.

Reply:
@prankul88 can you please show more detail for why `KUBECONFIG` was not needed? If did not set `KUBECONFIG`, how can I use `kubectl` command?

Reply:
@gyliu513 For reference https://github.com/kubernetes-sigs/kind/issues/1060

Reply:
@gyliu513 I too have been looking at how kind manages to avoid exporting KUBECONFIG. Will share details.

Reply:
There's a default value for kubeconfig `~/.kube/config` iirc. kind just mutates wherever your kubeconfig is pointing to add an additional context.  Minikube, docker desktop kubernetes, and some of our internal tooling uses a similar method.

Reply:
Good to know, thanks @prankul88 and @b2jrock